### DISCUSSION TITLE
Is the carousel touch/drag compatible?

### DISCUSSION THREAD

**Discussion by stanlrt** [chakra-ui/ark#2615] [Q&A] [answered]
This is the very first Zag/Ark component I'm using, so I might just have missed something.

Or is it not possible in its current beta state? If not, is it by chance planned? In the meantime I'll just use scroll-snap

**Comment by segunadebayo** [comment, marked as answer]
Not yet. We're looking to improve the carousel in the coming weeks.

The goal is to leverage scroll snap and reduce the overall reliance on JS for the Carousel feature. Of course, we'll add touch support as well.

Please use scroll snap + overflow auto for now.
